salt/metalk8s/kubernetes/apiserver/installed.sls

```diff
@@ -108,7 +108,7 @@ Create kube-apiserver Pod manifest:
           - --etcd-cafile=/etc/kubernetes/pki/etcd/ca.crt
           - --etcd-certfile=/etc/kubernetes/pki/apiserver-etcd-client.crt
           - --etcd-keyfile=/etc/kubernetes/pki/apiserver-etcd-client.key
-          - --etcd-servers=https://{{ pillar.registry_ip }}:2379
+          - --etcd-servers=https://{{ grains.metalk8s.control_plane_ip }}:2379
```

Comment:
This may work now (and so does the old approach), but we'll need to change this at some point in time to point to the actual etcd cluster addresses. Could you file a tracking ticket?

Reply:
Yep, issue created: https://github.com/scality/metalk8s/issues/1074